images/builders/mlflow/mlflow/mlflow_run.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+import argparse
+import mlflow
+import os
+from mlflow.projects.utils import load_project
+
+
+def run_mlflow_project(workdir: str, entrypoint: str, experiment: str, artifact_subpath: str, save_result_to_file: str) -> None:
+    """Run an MLproject entrypoint with arguments read from
+    FuseML workflow input parameters and save the result
+    artifact URL into the specified file.
+
+    Args:
+        workdir (str): path to the directory where the MLproject file resides
+        entrypoint (str): MLFlow project entrypoint to execute
+        experiment (str): MLFlow experiment name
+        artifact_subpath (str): path relative to the the MLFlow project run output
+                                artifact storage location to return as result
+        result_file (str): location of file where to save the URL of the output artifact
+    """
+    import os.path
+    if not os.path.isfile(os.path.join(workdir, "MLproject")):
+        print(f"Could not find MLproject file at location '{os.path.abspath(workdir)}'")
+        exit(1)
+    project = load_project(workdir)
+    if entrypoint not in project._entry_points:
+        print(f"Entrypoint '{entrypoint}' not found in MLproject definition file")
+        exit(1)
+    ep = project.get_entry_point(entrypoint)
+    parameters={}
+
+    # This section looks for any entrypoint arguments configured in the MLproject
+    # for the indicated entrypoint and attempts to match them to workflow step input
+    # parameters.
+    #
+    # If the user configures a workflow input parameter with the same name as the
+    # entrypoint argument, its value will be passed to the MLFlow project run.
+    for ep_param in ep.parameters:
+        v = os.environ.get("FUSEML_"+ep_param.upper())
+        if v is not None:
+            parameters[ep_param] = v
+
+    print(f"Launching '{entrypoint}' MLFlow entrypoint...")
```

Comment:
While you have some log output already, could you add the information about the parameters you're using?

Reply:
done